doc/release-notes/4.17-release-notes.md

```diff
@@ -0,0 +1,86 @@
+
+- As a user, I can launch and utilize external tools that allow me to work across the code, data, and other files in a dataset. 
+- As a developer, I can build external tools to verify reproducibility or allow computation.
+- As an installation administrator, I can make a quick configuration change to provide a better experience for my installation's users.
+
+## Notes for Dataverse Installation Administrators
+
+### Configurable User Session Timeout
+
+Idle session timeout for logged-in users has been made configurable in this release. 
+The default is now set to 8 hours (this is a change from the previous default value of 24 hours).
+If you want to change it, set the setting :LoginSessionTimeout to the new value *in minutes*. 
+For example, to reduce the timeout to 4 hours:
+
+    curl -X PUT -d 240 http://localhost:8080/api/admin/settings/:LoginSessionTimeout
+
+Once again, this is the session timeout for *logged-in* users only. For the anonymous sessions the sessions are set to time out after the default ``session-timeout`` value (also in minutes) in the web.xml of the Dataverse application, which is set to 10 minutes. You will most likely not ever need to change this, but if you do, configure it by editing the web.xml file. 
+
+### Flexible Solr Schema, optionally reconfigure Solr
+
+With this release, we moved all fields in Solr search index that relate to the default metadata schemas from `schema.xml` to separate files. Custom metadata block configuration of the search index can be more easily automated that way. For details, see admin/metadatacustomization.html#updating-the-solr-schema.
+
+This is optional, but all future changes will go to these files. It might be a good idea to reconfigure Solr now or be aware to look for changes to these files in the future, too. Here's how:
+
+1. You will need to replace or modify your `schema.xml` with the recent one (containing XML includes)
+2. Copy `schema_dv_mdb_fields.xml` and `schema_dv_mdb_copies.xml` to the same location as the `schema.xml`
+3. A re-index is not necessary as long no other changes happened, as this is only a reorganization of Solr fields from a single schema.xml file into multiple files.
+
+In case you use custom metadata blocks, you might find the new `updateSchemaMDB.sh` script beneficial. Again,
+see http://guides.dataverse.org/en/4.17/admin/metadatacustomization.html#updating-the-solr-schema
+
+### New Database Settings
+
+[:LoginSessionTimeout](http://guides.dataverse.org/en/4.17/installation/config.html#loginsessiontimeout) controls the session timeout (in minutes) for logged-in users.
+
+## Notes for Tool Developers and Integrators
+
+### Dataset Level Explore Tools
+
+Tool developers can now define the scope of an external tool as file or dataset. This will allow tools to appear for selection at the dataset or file level. For more information, check out the Guides section on <a href=http://guides.dataverse.org/en/develop/api/external-tools.html/>External Tools</a>. 
```

Comment:
```suggestion
Tool developers can now define the scope of an external tool as file or dataset. This will allow tools to appear for selection at the dataset or file level. For more information, check out the Guides section on <a href=http://guides.dataverse.org/en/4.17/api/external-tools.html/>External Tools</a>.
```